pytest_rts/utils/db.py

```diff
@@ -1,4 +1,5 @@
 """This module contains code for database actions"""
+# pylint: disable=too-many-public-methods
```

Comment:
Ahem. Pylint warnings are there not to disable them, but to refactor the things when they get out of control. :)

Reply:
Any tips on what to do with the `DatabaseHelper` class to make it fit to the pylint standard?

Reply:
First thing that comes to mind (not necessarily correct):
- make a context manager
- create classes MappingHelper and TestHelper, both should take connection as a constructor parameter and expose relevant methods/properties
- when `with database_helper() as db` is executed, make sure that `db.mappings` and `db.tests` are instances of MappingHelper and TestHelper

Something along the lines
```
@contextmanager
def database_helper():
    ...

...
with database_helper() as db:  # db is an instance of, say, DatabaseHelper
    ...
    print(db.tests.existing)  # db.tests is an instance of, say, TestsHelper
```

This is the first thought and not very structured, but maybe it gives you some idea.

Reply:
I played with the idea of creating some context manager but felt like I was making more of a mess because almost every file had to be changed. I now moved some methods from `db.py` to evaluation code because that's were they seem to belong and now the pylint doesn't complain. Maybe the context manager thing could be done in a different PR.